build/codegen/codegen.sh

```diff
@@ -14,29 +14,37 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-GROUP_VERSIONS="ceph.rook.io:v1"
+set -o pipefail
+set -o nounset
+set -o errexit
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 
-scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+base_dir="$(cd "${script_dir}/../.."  && pwd)"
+
+boilerplate="${base_dir}"/build/codegen/header.txt
+source "${CODE_GENERATOR}/kube_codegen.sh"
 
 # CODE GENERATION
 # we run deepcopy and client,lister,informer generations separately so we can use the flag "--plural-exceptions"
 # which is only known by client,lister,informer binary and not the deepcopy binary
```

Comment:
This comment doesn't apply to the latest tooling, which doesn't use separate generators. Without it, it's just `# CODE GENERATION` which doesn't add value, so let's just nix this comment altogether

Reply:
done